guides/common/modules/ref_dns-options-for-network-configuration.adoc

```diff
@@ -2,28 +2,37 @@
 = DNS Options for Network Configuration
 
 --foreman-proxy-dns::
-  Enables DNS service.
+  Enables the DNS feature.
```

Comment:
```suggestion
Enables the DNS feature.
```
Any reason to have spaces at the beginning? I have not observed that it would affect the build in any way. This applies to the rest of the file as well.

Reply:
It traces back to 156d071a3e61d7a73f071917cfc4a8970d99a5c3 so no idea.

Reply:
If it's not necessary, remove it. IMHO asciidoc works without.

Please note that this does not have to happen within this PR.

Reply:
I agree, but let's remove those in a future PR. We may end up removing the appendix altogether if we properly document the nsupdate provider instead with concrete examples.

Reply:
I wouldn't personally remove it, it makes the code easier to read.

Reply:
I had the same thought. let's get this merged and deal with it in a future PR.